microsetta_public_api/repo/_beta_repo.py

```diff
@@ -0,0 +1,43 @@
+from microsetta_public_api.repo._base import DiversityRepo
+from microsetta_public_api.exceptions import UnknownID, InvalidParameter
+import numpy as np
+
+
+class BetaRepo(DiversityRepo):
+
+    def __init__(self, resources):
+        super().__init__(resources)
+        self._ids = {key: set(dm.ids) for key, dm in resources.items()}
```

Comment:
I think this is redundant with the `in` operator of `DistanceMatrix`

Reply:
Good catch. See https://github.com/biocore/microsetta-public-api/pull/68/commits/2546a560976b8ef1fbf04b1836cb64679b771dce